app/helpers/ops_helper/textual_summary.rb

```diff
@@ -141,6 +153,129 @@ def textual_vmdb_tables_most_wasted_space
+  def textual_subtenant
+    return nil if @record.all_subtenants.blank?
+
+    {:label => _('Child Tenants'), :value => subtenants_from_record}
+  end
+
+  def textual_project
+    return nil if @record.all_subprojects.blank?
+
+    {:label => _('Projects'), :value => subprojects_from_record}
+  end
+
+  def textual_catalog_items
+    cat_items_num = ServiceTemplate.with_tenant(@record.id).count # Number of relevant Catalog Items and Bundles
+    cat_items = {:label => _('Catalog Items and Bundles'),
+                 :value => cat_items_num,
+                 :icon  => 'ff ff-group'}
+
+    if cat_items_num.positive?
+      cat_items[:link] = url_for_only_path(:action => 'show', :id => @record.id, :display => 'service_templates')
+      cat_items[:title] = _('View the list of relevant Catalog Items and Bundles')
+      cat_items[:explorer] = true
+    end
+
+    cat_items
+  end
+
+  def textual_automate_domains
+    aedomains_num = MiqAeDomain.with_tenant(@record.id).count # Number of relevant Automate Domains
+    aedomains = {:label => _('Automate Domains'),
+                 :value => aedomains_num,
+                 :icon  => 'ff ff-group'}
+
+    if aedomains_num.positive?
+      aedomains[:link] = url_for_only_path(:action => 'show', :id => @record.id, :display => 'ae_namespaces')
+      aedomains[:title] = _('View the list of relevant Automate Domains')
+      aedomains[:explorer] = true
+    end
+
+    aedomains
+  end
+
+  def textual_providers
+    providers_num = ExtManagementSystem.with_tenant(@record.id).count # Number of relevant Providers
+    providers = {:label => _('Providers'),
+                 :value => providers_num,
+                 :icon  => 'ff ff-group'}
+
+    if providers_num.positive?
+      providers[:link] = url_for_only_path(:action => 'show', :id => @record.id, :display => 'providers')
+      providers[:title] = _('View the list of relevant Providers')
+      providers[:explorer] = true
+    end
+
+    providers
+  end
+
+  def groups_from_record
+    record_groups = []
+    @record.miq_groups.non_tenant_groups.sort_by { |group| group.name.downcase }.each do |g|
+      if role_allows?(:feature => "rbac_group_show")
+        record_groups.push(
+          :value    => g.name,
+          :title    => _("Click to view %{name} Group") % {:name => g.name},
+          :explorer => true,
+          :link     => url_for_only_path(:controller => 'ops', :action => 'tree_select', :id => "g-#{g.id}")
+        )
+      else
+        record_groups.push(:value => g.name)
+      end
+    end
+    record_groups
+  end
+
+  def subtenants_from_record
+    subtenants = []
+    @record.all_subtenants.each do |tenant|
+      subtenants.push(
+        :value    => tenant.name,
+        :title    => _("Click to view %{name} Tenant") % {:name => tenant.name},
+        :explorer => true,
+        :link     => url_for_only_path(:controller => 'ops', :action => 'tree_select', :id => "tn-#{tenant.id}")
+      )
+    end
+    subtenants
```

Comment:
@hstastna could we sort the tenants alphabetically, so they match the order in the tree?

Reply:
Thank you, Roman, I think, yes, we could. I will update the PR asap with this change.